easybuild/easyconfigs/o/openslide-python/openslide-python-1.1.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'openslide-python'
+version = '1.1.1'
+
+github_account = 'openslide'
+homepage = 'https://github.com/%(github_account)s/%(name)s'
```

Comment:
@lexming nit: since the homepage is only for documentation, I prefer to have the full url, so I can click on it :)

Reply:
Fixed :+1: